batch/build-batch-worker-image-startup.sh

```diff
@@ -6,6 +6,7 @@ apt-get install -y \
     apt-transport-https \
     ca-certificates \
     curl \
+    jq \
```

Comment:
Thought: we could store the metadata as JSON now instead of in a long list of metadata keys. That might make the batch worker start up code easier to follow. Also reduces the number of network requests when a worker starts.